core/src/com/unciv/logic/civilization/TechManager.kt

```diff
@@ -44,7 +45,20 @@ class TechManager {
         if (civInfo.isPlayerCivilization())
             techCost *= civInfo.getDifficulty().researchCostModifier
         techCost *= civInfo.gameInfo.gameParameters.gameSpeed.getModifier()
-        techCost *= 1 + (civInfo.cities.size -1 ) * 0.02f // each city increases tech cost by 2%, as per https://civilization.fandom.com/wiki/Science_(Civ5)
+        var techsResearchedKnownCivs = 0
+        civInfo.getKnownCivs().filter { it.isMajorCiv() }.forEach {
+            if (it.tech.isResearched(techName)) techsResearchedKnownCivs++
+        }
+        val undefeatedCivs= UnCivGame.Current.gameInfo.civilizations.filter { it.isMajorCiv() && !it.isDefeated() }.size
+        techCost /= 1 + techsResearchedKnownCivs / undefeatedCivs * 0.3f
```

Comment:
Please add references in comments, for both this and the worldSizeModifier

Reply:
ok, but i have not found a good english web to explain this, i am not sure whether adding Chinese web is a good idea or not.

Reply:
Okay, better than nothing if we're going to add features based on that source

Reply:
ok, but i have not found why it has no effort.